backend/backend/core/ollama.py

```diff
@@ -1,33 +1,53 @@
 import httpx
 
-from langchain_ollama import ChatOllama
+from pydantic.json_schema import JsonSchemaValue
+from typing import Literal, Union
 
-from utils.logger import logger
+from langchain_ollama import ChatOllama
 
 from config import settings
+from utils.logger import logger
 
 
 class OllamaBackend(object):
     def __init__(self):
         self._pull_models()
 
     def _pull_models(self):
-        logger.info(f"Pulling Ollama model '{settings.OLLAMA_MODEL}' at endpoint '{
-                    settings.OLLAMA_ENDPOINT}' ...")
+        logger.info(
+            f"Pulling Ollama model '{settings.OLLAMA_MODEL}' at endpoint '{
+                    settings.OLLAMA_ENDPOINT}' ..."
+        )
         with httpx.Client() as client:
             response = client.post(
                 f"{settings.OLLAMA_ENDPOINT}/api/pull",
-                json={"name": settings.OLLAMA_MODEL})
+                json={"name": settings.OLLAMA_MODEL},
+            )
             for chunk in response.iter_lines():
                 logger.info(f"Ollama: {chunk}")
         logger.info("Pulling Ollama model done")
 
     def get_chat_model(self):
-        return ChatOllama(model=settings.OLLAMA_MODEL,
-                          base_url=settings.OLLAMA_ENDPOINT)
-
-    def get_chat_model_json(self):
-        return ChatOllama(model=settings.OLLAMA_MODEL,
-                          base_url=settings.OLLAMA_ENDPOINT,
-                          format="json",
-                          temperature=0.1)
+        return ChatOllama(
+            model=settings.OLLAMA_MODEL, base_url=settings.OLLAMA_ENDPOINT
+        )
+
+    def get_chat_model_json(
+        self, format: Union[Literal["", "json"], JsonSchemaValue] = "json"
```

Comment:
The `format` parameter in the `get_chat_model_json` method is annotated with a type that includes `JsonSchemaValue`, but there is a comment indicating that this is not valid according to the langchain documentation. This could lead to runtime errors or unexpected behavior.